### PR TITLE
fix: sensor views shown without PSLab device

### LINKED ISSUE #1283: Redesign Sensors section
**Actual Behaviour**

When no device is connected, cannot view sensor UI views

**Expected Behaviour**

Even when there is no sensor or device is connected, app should display the inside view of UI

**Steps to reproduce it**

Observe sensors instrument

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixes #1283 

**Changes**: 
- Made changes to view UI inside sensors even if PSLab device is not connected.

**Screenshot/s for the changes**: 
> Not available

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [X] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [X] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [X] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [X] My code does not contain any extra lines or extra spaces
- [X] I have requested reviews from other members

**APK for testing**: 
[sensors.apk.zip](https://github.com/fossasia/pslab-android/files/2268783/sensors.apk.zip)

### CODE REVIEW

File: app/src/main/java/org/pslab/sensors/SensorADS1115.java [hunk 1]
```diff
@@ -188,7 +188,10 @@ private void setSensorDock() {
         playPauseButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (play) {
+                if (play & scienceLab.isConnected()) {
```
Comment (abhinavraj23) [inline app/src/main/java/org/pslab/sensors/SensorADS1115.java]: @CloudyPadmal Is there a specific reason to use single & operator and not && , I think && should be  better.

Comment (CloudyPadmal) [reply app/src/main/java/org/pslab/sensors/SensorADS1115.java]: Ahh yes! I'll make the changes! :+1:

**Comment by cweitat** [comment]
@CloudyPadmal there's some codacy issues.

**Comment by CloudyPadmal** [comment]
Now Codacy build is passing!

**Pull request merged**